lib/routes/verify.js

```diff
@@ -25,15 +25,7 @@ module.exports = {
   },
   handler: function verify(req, reply) {
     token.verify(req.payload.token).then(function(info) {
-      if (req.payload.email) {
```

Comment:
I think it would be worth making `email` be `Joi.boolean().optional()` in the payload schema above, and changing this line to log a warning whenever `req.payload.email !== undefined`.  This would help us confirm that all services have indeed stopped sending an explicit `email: false` in their requests.

Reply:
@rfk done